src/Hng.Web/Program.cs

```diff
@@ -34,6 +37,29 @@
 builder.Services.AddScoped<IUserService, UserService>();
 builder.Services.AddScoped<SeederService>();
 builder.Services.Configure<Microsoft.AspNetCore.Http.Json.JsonOptions>(options => options.SerializerOptions.ReferenceHandler = ReferenceHandler.IgnoreCycles);
+
```

Comment:
Stop putting large spaces in files. You will cause a lot of conflicts. A single space is okay.